### PR TITLE
Added CloudFront gotchas about 2048 bit ssl cert

### PR DESCRIPTION
**Pull request by Bizzelicious**

### CODE REVIEW

**Comment by jlevy** [comment]
I guess it's reasonable to have this in both CloudFront and in load balancing (but only once in the latter). Can we update this one once #281 is done to match the language/style?

**Comment by jlevy** [comment]
Thanks again!

File: README.md [hunk 1]
```diff
@@ -1332,6 +1332,7 @@ CloudFront
 
 -	🔸If using S3 as a backing store, remember that the endpoints for website hosting and for general S3 are different. Example: “bucketname.s3.amazonaws.com” is a standard S3 serving endpoint, but to have redirect and error page support, you need to use the website hosting endpoint listed for that bucket, e.g. “bucketname.s3-website-us-east-1.amazonaws.com” (or the appropriate region).
 -   🔸By default, CloudFront will not forward HTTP Host: headers through to your origin servers. This can be problematic for your origin if you run multiple sites switched with host headers. You can [enable host header forwarding](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/RequestAndResponseBehaviorCustomOrigin.html#request-custom-headers-behavior) in the default cache behavior settings.
+-   🔸If you want to use a self-obtained SSL certificate with Cloudfront. Make sure it's 2048 bits since Cloudfront only support 2048 bits SSL certificates. [Ongoing discussion](https://forums.aws.amazon.com/thread.jspa?threadID=148783)
```
Comment (ThanosBaskous) [inline README.md]: Grammar / phrasing suggestion:

`Custom SSL certificates used with CloudFront can only be up to 2048 bits. See [ongoing discussion](https://forums.aws.amazon.com/thread.jspa?threadID=148783).`

Comment (jlevy) [inline README.md]: I'm OK with this phrasing or the one I suggested in #281, but also they should match.

**Comment by Bizzelicious** [comment]
Updated phrasing according to #281

**Comment by jlevy** [comment]
👍 Thanks for the addition!

**Pull request merged**